src/Mod/TechDraw/App/DrawUtil.cpp

```diff
@@ -1899,6 +1899,18 @@ bool DrawUtil::isCenterLine(App::DocumentObject* owner, std::string element)
     return false;
 }
 
+//! convert a filespec (string) containing '\' to only use '/'.
+//! prevents situation where '\' is interpreted as an escape of the next character in Python
+//! commands.
+std::string DrawUtil::cleanFilespecBackslash(const std::string& filespec)
+{
+    std::string forwardSlash{"/"};
+    boost::regex rxBackslash("\\\\");    //this rx really means match to a single '\'
```

Comment:
Consider using a "raw" string literal for regexes, e.g.:
```suggestion
    boost::regex rxBackslash(R"\\");    //this rx really means match to a single '\'
```